calculator/lexer.py

```diff
@@ -1,4 +1,65 @@
-def lex(s):
+ops = ["+", "-", "*", "/"]
+ops_without_subtraction = ["+", "*", "/"]
```

Comment:
Constant names should be all-caps (style recommendation), so please change `ops` and `ops_without_subtraction` to `OPS` and `OPS_WITHOUT_SUBTRACTION` respectively

Reply:
👍